src/Sulu/Component/Content/SmartContent/QueryBuilder.php

```diff
@@ -258,6 +277,33 @@ private function buildDatasourceWhere()
     }
 
     /**
+     * Returns the where part for the audience targeting.
+     *
+     * @param string $locale
+     *
+     * @return string|void
+     */
+    private function buildAudienceTargeting($locale)
+    {
+        $targetGroup = $this->targetGroupEvaluator->evaluate();
```

Comment:
It's the real evaluation, and returns the result of the evaluation, which is a target group.